zsh/prompt.zsh

```diff
@@ -19,3 +19,7 @@ GIT_PROMPT_DIRTY="%{$fg_bold[yellow]%}*%{$reset_color%}"
 
 setopt PROMPT_SUBST
 PS1='%{$fg[green]%}%3~$(git_prompt_info) %{$reset_color%}$ '
+
+if (( ${+SSH_CLIENT} )) || (( ${+SSH_TTY} )) || (( ${+SSH_CONNECTION} )); then 
```

Comment:
@paydaylight, I see that you use quite complex syntax here:

`${+SSH_CLIENT}` resolves to `1` if variable is set and `0` if it's not ([zsh parameters expansion](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Parameter-Expansion))
`(( 1 ))` returns 0 which is treated as true by if.

I suggest to use `[[ -v SSH_CLIENT ]]` which is simpler: http://zsh.sourceforge.net/Doc/Release/Conditional-Expressions.html#Conditional-Expressions.

Reply:
Oh, I see. Changed it in a new commit.